.storybook/preview.ts

```diff
@@ -20,7 +31,6 @@ addParameters({
     }
   },
   docs: {
-    page: titlelessDocsPage,
```

Comment:
@jcfranco can you help me fix this for storybook 6?

Reply:
👍